src/main/java/com/stripe/model/SubscriptionItem.java

```diff
@@ -71,6 +71,15 @@
   @SerializedName("plan")
   Plan plan;
 
+  /**
+   * The price object defines the unit cost (or tiered cost) for both recurring and one-time prices.
+   * For example, you might have a &lt;currency&gt;5&lt;/currency&gt;/month recurring price for
```

Comment:
There are encoding issues here and likely in similar docstrings that we need to look into

Reply:
Same as above, not resolved